lms/templates/ccx/schedule.underscore

```diff
@@ -71,27 +68,32 @@
             </button></td>
           </tr>
           <% _.each(child.children, function(subchild) { %>
-            <tr class="vertical" data-dapth="3"
+            <tr class="vertical" data-depth="3"
                 data-location="<%= chapter.location %> <%= child.location %> <%= subchild.location %>">
               <td class="unit">&nbsp;
                  <span class="sr"><%- gettext('Unit') %>&nbsp;</span>
                  <%= subchild.display_name %>
               </td>
-              <td class="date start-date">
-                <button class="ccx-button-link">
+              <td class="date start-date no-link">
+                <% if (subchild.start) { %>
                   <%= subchild.start %>
-                  <span class="sr"><%- gettext('Click to change') %></span>
-                </button>
+                <% } else { %>
+                  <%- gettext('Unscheduled') %>
+                <% } %>
               </td>
-              <td class="date due-date">
-                <button class="ccx-button-link">
+              <td class="date due-date no-link">
+                <% if (subchild.due) { %>
                   <%= subchild.due %>
-                  <span class="sr"><%- gettext('Click to change') %></span>
+                <% } else { %>
+                  <%- gettext('Unscheduled') %>
+                <% } %>
+              </td>
+              <td>
+                <button class="remove-unit ccx-button-link" aria-label="<%- gettext('Remove unit') %> <%= subchild.display_name %>">
```

Comment:
Translate whole phrases, not fragments. In this case, the whole phrase is "Remove unit &lt;name&gt;" -- other languages may swap the order of the sentence structure. You should do this instead:
```
aria-label="<%- interpolate(
    gettext('Remove unit %(unitName)s'), {unitName: subchild.display_name}, true
) %>"
```
The `interpolate()` function is documented here: https://docs.djangoproject.com/en/1.8/topics/i18n/translation/#interpolate